src/main/kotlin/no/nav/familie/oppdrag/repository/OppdragProtokoll.kt

```diff
@@ -1,23 +1,31 @@
 package no.nav.familie.oppdrag.repository
 
 import com.fasterxml.jackson.databind.ObjectMapper
+import no.nav.familie.oppdrag.domene.OppdragId
+import no.nav.familie.oppdrag.domene.id
 import no.trygdeetaten.skjema.oppdrag.Oppdrag
 import org.springframework.data.annotation.Id
 import org.springframework.data.relational.core.mapping.Column
 import java.time.LocalDateTime
+import javax.persistence.IdClass
 
-data class OppdragProtokoll(@Id val serienummer: Long = 0,
-                            val id : String,
+@IdClass(OppdragId::class)
+data class OppdragProtokoll(@Id val fagsystem : String,
+                            @Id val fødselsnummer : String,
```

Comment:
Ser vi har litt overlapp i våre PR-er, vi kan jo sette oss sammen og finne ut av det. Men jeg tror det er best om denne kolonnen heter personident eller lignende, for vi aksepterer D-nummer også